dataplatform_keycloak/teams_client.py

```diff
@@ -140,6 +147,33 @@ def get_team_members(self, team_id, realm_role=None):
             raise TeamsServerError
         return members
 
+    def update_team(self, team_id, name, attributes):
+        team = self.get_team(team_id)
+
+        if name:
+            team["name"] = team_name_to_group_name(name)
+
+        for team_attr in attributes.keys() & TEAM_ATTRIBUTES:
+            group_attr = team_attribute_to_group_attribute(team_attr)
+
+            if value := attributes[team_attr]:
+                team["attributes"][group_attr] = [value]
```

Comment:
Er det potensielt forvirrende at vi kun tillater  enkeltverdier for attributter i patch, men returnerer en liste i output? Vi snakket vel om dette tidligere, men husker ikke hva vi landet på.